File: Online2/C/q.py
n = int(input())
m = input().split()
for i in range(len(m)):
    m[i] = int(m[i])
k = 0
for x in range(len(m)):
    if m[x] < 30 or m[x] > 120:
        k += 1

ko = (k*100)/n
if ko < 10 :
    print("good")
elif ko >= 10 and ko <= 50 :
    print("normal")
else :
    print("bad")
        

File: Online2/C/q.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n, ans = 0;
    cin >> n;
    
    for(int i=1; i<=n; i++)
    {
        int x; cin >> x;
        if(x < 30 || x > 120) ans++;
    }
    
    if(ans*2 > n)      cout << "bad";
    else if(ans*10 > n) cout << "normal";
    else               cout << "good";
}
